pl_bolts/models/self_supervised/fixmatch/networks.py

```diff
@@ -0,0 +1,205 @@
+import math
+from copy import deepcopy
+
+import torch
+import torch.nn as nn
+from torch.nn import BatchNorm2d
+
+"""
+    As in the paper, the wide resnet only considers the resnet of the pre-activated version,
+    and it only considers the basic blocks rather than the bottleneck blocks.
+"""
```

Comment:
```suggestion
# As in the paper, the wide resnet only considers the resnet of the pre-activated version,
# and it only considers the basic blocks rather than the bottleneck blocks.
```